### DISCUSSION TITLE
Monopole name or identity in madanalysis

### DISCUSSION THREAD

**Discussion by Sidra1618** [MadAnalysis/madanalysis5#227] [Q&A]
Hi,
I have imported madgraph UFO model and created LHE files successfully.
Now we are unable to find name of monopole in madanalysis. What should exactly the name of monopole in madanalysis?

**Comment by BFuks** [comment]
Hi,

Please be more specific about what you have done and what you aim to do. We need more information to be able to help you. Note that there is no predefined monopole in MA5 so that  you need to define that particle first (i.e. by assigning a label to it through its PDG code).

Cheers,

Benjamin
